domains/bypass-rn.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Gauravisdev",
+           "email": "",
+           "discord": "1119494339735867422",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.SQ4-6TeOEf_gbtHEk33W3kjeZMy1UunlYcI37AuA5T5UWEVIPwz5pBNm_6dCkfWZlRBWjdhjLix9hMeVdUAv5eEfXp-isNMFtbsVTJTX8yTE6u4W7S1j_ad5vO3qVdgOCtIEZqQiCPmcss1_CUELdeuyZHdim8ZJMcCdywZgfonHXzh-_G68-jsYHLcU0bubUKjjlyWIFrilx6fl-drTLgREXrZw1mDaf14qZtmajGl5JMs3u7ogb-vVzOKRfd6P8fsu626C_ZQiV_RVJUPTxuO4KIFj7gM2dzxJnz4kZQcbRzH-_Ygx60KnSEiFGA9EC0nRJ3K0zocdRHSxI9tlZg.2tNbv94A6KE2rJuCSLMKsQ.taFiZJVZ3Z7xWfnTxEZKaiZYK9F6-Ya4mA8cBzQyNRUj_1TwNdHHElDS52-2lCnm_fzL__jqFP-QYREfGNaI-yvvWsschr8SDVJbjxkD4SQ.Paqhz6YUugrdXlio7LhRfw"
+        },
+    
+        "record": {
+            "CNAME": "gaurav-sto.onrender.com"
```

Comment:
You hsould complete your website, i only see a very simple text with a upload button that doesn't seem to work